fonts/petaluma/OFL-FAQ.txt

```diff
@@ -365,5 +365,5 @@ D: Designer - original Roman glyphs
 Fontstudio.org is an not-for-profit design group whose purpose is...
 Foobar.org is a distributed community of developers...
 Company.com is a small business who likes to support community designers...
-University.edu is a renowed educational institution with a strong design department...
+University.edu is a renowned educational institution with a strong design department...
```

Comment:
Taken from https://github.com/steinbergmedia/petaluma, so wouzld need to get fixed there (too/first)

Reply:
https://github.com/musescore/MuseScore/pull/11566#discussion_r867539770